7_react/react1/week2/my-app/src/Header/Header.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import "./Header.css"
+function Header(){
+    return(
+    <>
```

Comment:
We don't need the extra `<>` brackets here, since the code is sitting inside a `<header>` HTML tag already